src/hooks/sounds.tsx

```diff
@@ -260,5 +274,5 @@ export const useCustomSound = (
     [play, spriteName, hasSoundConfig, hasValidSpriteConfig, onend]
   );
 
-  return [playSprite, { ...exposedData, play }];
+  return [playSprite, { ...exposedData, play }, isCustomSoundsLoaded];
```

Comment:
As mentioned in an earlier comment, it would be better to return named keys in an object here now, rather than a positional array like this.